snippets/go/getting_started.go

```diff
@@ -71,3 +71,18 @@ func FetchBalance() {
 	}
 	// ANCHOR_END: fetch-balance
 }
+
+// ANCHOR: logging
+type SdkLogStream struct{}
+
+func (SdkLogStream) Log(l breez_sdk.LogEntry) {
+	log.Printf("Received log [%v]: %v", l.Level, l.Line)
+}
+
+func GettingStartedLogging() {
+	err := breez_sdk.SetLogStream(SdkLogStream{})
+	if err != nil {
+		log.Fatalf("SetLogStream failed: %#v", err)
```

Comment:
Needs to be reformatted according to CI.

Reply:
Fixed [ba0b87c](https://github.com/breez/breez-sdk-docs/pull/170/commits/ba0b87ccdf99fa2a1cd2373e6781a1e5d0ce9729)